test/TopupEUReTest.t.sol

```diff
@@ -43,26 +45,28 @@ contract TopupTest is BaseFixture {
         delayModule.executeNextTx(EURE, 0, payload, Enum.Operation.Call);
 
         (bool canExec, bytes memory execPayload) = roboModule.checker();
+        (bytes memory dataWithoutSelector, bytes4 selector) = _extractEncodeDataWithoutSelector(execPayload);
 
         assertTrue(canExec);
-        assertEq(bytes4(execPayload), SAFE_TOP_UP_SELECTOR);
+        assertEq(selector, ADJUST_POOL_SELECTOR);
 
         uint256 initialEureBal = IERC20(EURE).balanceOf(GNOSIS_SAFE);
 
         vm.prank(TOP_UP_AGENT);
-        (bool success, bytes memory data) = address(roboModule).call(execPayload);
-        assertTrue(success);
+        (RoboSaverVirtualModule.PoolAction _action, address _card, uint256 _amount) =
+            abi.decode(dataWithoutSelector, (RoboSaverVirtualModule.PoolAction, address, uint256));
```

Comment:
aah you decode here! so lets assert `_action` is a pool withdrawal here?

Reply:
done in 5f1fcea904ca397fb0a2d7390dd29c269adb5241